app/client/src/plugins/Linting/constants.ts

```
@@ -1,38 +1,85 @@
+      globals: globalData,
+      loopfunc: true,
+    } as LintOptions;
+  } else {
+    const eslintGlobals: Record<string, "writable" | "readonly"> = {
+      setTimeout: "readonly",
+      clearTimeout: "readonly",
+      console: "readonly",
+    };
+
+    for (const key in globalData) {
+      if (globalData.hasOwnProperty(key)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)